docs/source/pillars/data_management.md

```diff
@@ -149,6 +149,17 @@ The ability of the TRE operator to ensure outputs are safely published and share
     Results will be produced which address the project's purpose, some of which may be outputs that are removed from the TRE.
     Understanding what these outputs are likely to be and their sensitivity as early as possible will help prepare for their processing and publication.
   - Recommended
+* - 3.3.3.
+  - You must have a process for disclosure control that ensures outputs are safe.
+  - This process should define expected risks and how to mitigate them.
+    All TRE outputs must be subject to this process.
+    You might choose to follow existing guidelines, for example around statistical disclosure.
+  - Mandatory
+* - 3.3.4.
+  - You must have a process for assigning responsibility for output checking.
+  - A named person or team should be given responsibility for checking outputs. This person or team must follow your disclosure control process.
```

Comment:
```suggestion
  - A named person or team should be given responsibility for checking outputs.
    This person or team must follow your disclosure control process.
```

Reply:
The SACRO project is developing semi-automated tools for output checking so it would be good to make reference to that as well. The responsible person will be responsible for automated checks as well.